worlds/jakanddaxter/client/ReplClient.py

```diff
@@ -41,10 +44,14 @@ class JakAndDaxterReplClient:
     item_inbox: Dict[int, NetworkItem] = {}
     inbox_index = 0
 
-    my_item_name: Optional[str] = None
-    my_item_finder: Optional[str] = None
-    their_item_name: Optional[str] = None
-    their_item_owner: Optional[str] = None
+    @dataclass
+    class JsonMessageData:
```

Comment:
yeah it shouldn't cause any issue. I kept it inside the ReplClient class because I didn't see it being used outside of the ReplClient class, so it didn't need to be visible outside of it, but if you prefer it outside for style/clarity it shouldn't change much

Reply:
It's entirely a personal thing - in general I try to avoid nested class definition, even if they make perfect sense in a language like Python that doesn't have access modifiers (public/private/etc). But maybe that's just crazy talk and there's no reason for me to work like that. I'll leave it up to you if you want to change it or not, just let me know either way.

Reply:
Would you rather have it in the same file but not in the ReplClient class or in its own little file to import in ReplClient ?

Reply:
Same file separate classes is fine by me. MemoryReader.py has this setup as well.

Reply:
Aight, it does look a bit cleaner compared to having it kinda in the middle of the the ReplClient class member initialization so I moved it just outside ReplClient